gtsfm/data_association/data_assoc.py

```diff
@@ -117,10 +117,10 @@ def run(
         track_cheirality_failure_ratio = num_tracks_w_cheirality_exceptions / len(tracks_2d)
 
         # pick only the largest connected component
-        result_data = triangulated_data.select_largest_connected_component()
+        connected_data = triangulated_data.select_largest_connected_component()
 
         mean_3d_track_length, median_3d_track_length, track_lengths_3d = SfmResult(
-            result_data, total_reproj_error=float("Nan")
+            triangulated_data, total_reproj_error=float("Nan")
```

Comment:
this should be `connected_data`?

Reply:
I think we could go either way. We could save just the DA stats for the largest connected component, or for the whole DA output.

The advantage of the former is that it gives us a 1:1 correspondence with the BA stats. But the latter is more representative of how DA did on average.

Perhaps the former is best? @ayushbaid @akshay-krishnan

Reply:
Sorry missing context here, what are connected data and triangulated data?

Reply:
So after data association, we make have a disconnected graph. So we take the largest connected component of cameras.

So we could report DA stats before or after pruning to largest CC>